api/v1alpha1/xlinecluster_types.go

```diff
@@ -47,6 +49,50 @@ type XlineClusterList struct {
 	Items           []XlineCluster `json:"items"`
 }
 
+// XlineArgs
+// +k8s:openapi-gen=true
+type XlineArgs struct {
+	IsLeader                    bool    `json:"isLeader,omitempty"`
+	JaegerOffline               bool    `json:"jaegerOffline,omitempty"`
+	JaegerOnline                bool    `json:"jaegerOnline,omitempty"`
+	JaegerLevel                 bool    `json:"jaegerLevel,omitempty"`
+	ClientUseBackoff            bool    `json:"clientUseBackoff,omitempty"`
+	AuthPrivateKey              *string `json:"authPrivateKey,omitempty"`
+	AuthPublicKey               *string `json:"authPublicKey,omitempty"`
+	JaegerOutputDir             *string `json:"jaegerOutputDir,omitempty"`
+	LogFile                     *string `json:"logFile,omitempty"`
+	LogRotate                   *string `json:"logRotate,omitempty"`
+	LogLevel                    *string `json:"logLevel,omitempty"`
+	HeartbeatInterval           *string `json:"heartbeatInterval,omitempty"`
+	ServerWaitSyncedTimeout     *string `json:"serverWaitSyncedTimeout,omitempty"`
+	RetryTimeout                *string `json:"retryTimeout,omitempty"`
+	RpcTimeout                  *string `json:"rpcTimeout,omitempty"`
+	BatchTimeout                *string `json:"batchTimeout,omitempty"`
+	ClientWaitSyncedTimeout     *string `json:"clientWaitSyncedTimeout,omitempty"`
+	ClientProposeTimeout        *string `json:"clientProposeTimeout,omitempty"`
+	ClientInitialRetryTimeout   *string `json:"clientInitialRetryTimeout,omitempty"`
+	ClientMaxRetryTimeout       *string `json:"clientMaxRetryTimeout,omitempty"`
+	GcInterval                  *string `json:"gcInterval,omitempty"`
+	RangeRetryTimeout           *string `json:"rangeRetryTimeout,omitempty"`
+	CompactTimeout              *string `json:"compactTimeout,omitempty"`
+	SyncVictimsInterval         *string `json:"syncVictimsInterval,omitempty"`
+	WatchProgressNotifyInterval *string `json:"watchProgressNotifyInterval,omitempty"`
+	CurpDir                     *string `json:"curpDir,omitempty"`
+	CompactSleepInterval        *string `json:"compactSleepInterval,omitempty"`
+	AutoCompactMode             *string `json:"autoCompactMode,omitempty"`
+	AutoPeriodicRetention       *string `json:"autoPeriodicRetention,omitempty"`
+	AutoRevisionRetention       *string `json:"autoRevisionRetention,omitempty"`
+	InitialClusterState         *string `json:"initialClusterState,omitempty"`
+	RetryCount                  *int    `json:"retryCount,omitempty"`
+	BatchMaxSize                *int    `json:"batchMaxSize,omitempty"`
+	FollowerTimeoutTicks        *int    `json:"followerTimeoutTicks,omitempty"`
+	CandidateTimeoutTicks       *int    `json:"candidateTimeoutTicks,omitempty"`
+	LogEntriesCap               *int    `json:"logEntriesCap,omitempty"`
+	CmdWorkers                  *int    `json:"cmdWorkers,omitempty"`
+	CompactBatchSize            *int    `json:"compactBatchSize,omitempty"`
+	Quota                       *int    `json:"quota,omitempty"`
```

Comment:
please modify the json field name in the annotation, for example,  modify `json:"jaegerOffline,omitempty"` to `json:"jaeger-offline,omitempty"`.

Reply:
Need to maintain the same specification as k8s yaml.

Reply:
The `bootstrapArgs` meets the specification of k8s yaml